h5p-bildetema-words-topic-image/semantics.json

```diff
@@ -216,5 +216,17 @@
     "type": "boolean",
     "default": false,
     "importance": "low"
+  },
+  {
+    "label": "Show only topic image",
+    "name": "showOnlyTopicImage",
+    "type": "boolean",
+    "default": false
+  },
+  {
+    "label": "Is published",
```

Comment:
Could have "Publish" as `label`, maybe. To simplify the label for the end users. `name` can be as it is